### ISSUE TITLE
Code review doc - is this quote appropriate?

### ISSUE THREAD

**Issue by reubengt** [foundersandcoders/coursebook#576]
<img width="529" alt="Screenshot 2022-01-26 at 17 10 19" src="https://user-images.githubusercontent.com/42008435/151212220-c1d265a4-9286-4eca-af55-f09172beeaa4.png">

just going through everything in the handbook.
I really dislike this quote in the code review checklist, it just reads too aggressively to me and doesn't seem like a great way to start off a checklist about positive, constructive feedback.

**Comment by oliverjam** [comment]
Yeah I like the sentiment but not the specific wording. Probably better to just get rid of it. The idea that code is for humans not computers isn't that important

**Comment by reubengt** [comment]
We can maybe rephrase this in our own words?

**Comment by oliverjam** [comment]
> "Write code for humans to read, not computers
> \- Wayne Gretsky"
> \- Michael Scott
